### PR TITLE
Add some pending tests and general updates

### PR DESCRIPTION
**Pull request by abelgardep**
Add some pending tests related with #2827

### CODE REVIEW

File: owncloudApp/src/main/java/com/owncloud/android/presentation/ui/sharing/fragments/ShareFileFragment.kt [hunk 1]
```diff
@@ -91,7 +91,7 @@ class ShareFileFragment : Fragment(), ShareUserListAdapter.ShareUserAdapterListe
     /**
      * List of private shares bound to the file
      */
-    private var privateShares: List<OCShare>? = null
+    private var privateShares: List<OCShare> = listOf()
```
Comment (davigonz) [inline owncloudApp/src/main/java/com/owncloud/android/presentation/ui/sharing/fragments/ShareFileFragment.kt:94]: 🚀

**Comment by davigonz** [comment]
Approved, please rebase before starting with QA

**Comment by jesmrec** [comment]
as CI is passing with all the tests included here, not sure if there are something else to test. I'd say no, but...

**Comment by abelgardep** [comment]
@jesmrec I added some pending tests, and updated some potential nullability problems related with shares. That would be the only thing to test here.

**Comment by jesmrec** [comment]
will take the final testing in regression stage.

**Pull request merged**